Gemfile

```diff
@@ -40,7 +40,7 @@ group :development, :test do
   gem 'launchy'
   gem 'factory_girl'
   gem 'valid_attribute'
-  gem 'shoulda-matchers', require: false
+  gem 'shoulda'
```

Comment:
This 'it should' behavior can be mirrored with Rspec 3's `expect` syntax. 

`it { is_expected.to validate_presence_of :email }`

This is fine for now!